src/features/Applet/Schedule/Calendar/EventWrapper/EventWrapper.tsx

```diff
@@ -1,4 +1,41 @@
+import { cloneElement, useEffect, useRef } from 'react';
+import { Box } from '@mui/material';
+
 import { EventWrapperProps } from './EventWrapper.types';
 
-export const EventWrapper = ({ event, children }: EventWrapperProps) =>
-  event.isHiddenInTimeView ? '' : children;
+export const EventWrapper = ({ event, children, components }: EventWrapperProps) => {
+  const emptyRef = useRef<HTMLElement>(null);
+  const childrenRef = useRef<HTMLElement>(null);
+  const isVisible = components.isAllDayEventsVisible?.visible;
+
+  useEffect(() => {
+    const parentElement =
+      emptyRef.current?.parentElement?.parentElement ||
+      childrenRef.current?.parentElement?.parentElement;
+
+    if (parentElement) {
+      for (let i = 0; i < parentElement.children.length; i++) {
```

Comment:
for (el of [...parentElement.children]) {
  statement
}

Reply:
done